plugins/CoreAdminHome/templates/whatIsNew.twig

```diff
@@ -42,19 +42,25 @@
  <script>
      function doNotShowAgain() {
         var ajaxRequest = new ajaxHelper();
-        ajaxRequest.addParams({module: 'API', method: 'CoreAdminHome.whatIsNewMarkAllChangesReadForCurrentUser'}, 'get');
+        ajaxRequest.addParams({
+            module: 'API',
+            method: 'CoreAdminHome.whatIsNewMarkAllChangesReadForCurrentUser',
+            format: 'json'
+        }, 'get');
         ajaxRequest.withTokenInUrl();
         ajaxRequest.setFormat('json');
+        ajaxRequest.setCallback(function() {
```

Comment:
This should prevent the request from being aborted before saving the new state.

But if the request, for whatever reason, fails, then nothing happens. We could add a `ajaxRequest.useCallbackInCaseOfError()` to always close the popover. Then it would only show again after the regular 24h delay, but perhaps be less confusing than just not closing the popover at all?

Reply:
added 👍